backend/de.metas.business/src/main/java-legacy/org/compiere/model/MOrder.java

```diff
@@ -488,10 +488,6 @@ public void setBPartner(final I_C_BPartner bp)
 			}
 		}
 		// set to first
-		if (getC_BPartner_Location_ID() == 0 && !locs.isEmpty())
```

Comment:
pls leave a comment that we are no longer setting to default location if that's not a ship location

Reply:
commented.